runtime/shiden/src/lib.rs

```diff
@@ -748,32 +756,39 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsWithSystem,
-    DappsStakingFixEraLength,
+    (InitRewardConfigSettings,),
 >;
 
 // Migration for fixing era length in dapps staking.
-pub struct DappsStakingFixEraLength;
+pub struct InitRewardConfigSettings;
```

Comment:
is the above comment still valid?

Reply:
Hmmm no, it's not.

This code will be deleted after the upgrade though.